coopr-docs/docs/source/guide/superadmin/docker-automator-plugin.rst

```diff
@@ -0,0 +1,119 @@
+..
+   Copyright © 2012-2015 Cask Data, Inc.
```

Comment:
I'm easy on this. If you think it's derivative, it's derivative and let's have both dates.